examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
+    {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory Upper Limit",
+      "description": "Maximum amount of memory the container can use.",
+      "value": "2000Mi"
+    },
```

Comment:
btw, this is why our image works when the container is resource limited and the docker hub image does not:
https://github.com/openshift/jenkins/blob/master/1/contrib/s2i/run#L16-L27

we set the java max heap appropriately based on the container memory constraints.